services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/mapper/testbeans/AbstractImmutable.java

```diff
@@ -29,6 +30,25 @@ public String attribute2() {
         return attribute2;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        AbstractImmutable that = (AbstractImmutable) o;
+
+        return Objects.equals(attribute2, that.attribute2);
```

Comment:
Nit: I think we've been using a different template for our other equals() implementations. The one that says 'intellij default'